### PR TITLE
IOTEDGE-1125 Update to ForgeOps 7.1

### PR DESCRIPTION
**Pull request by JacoJooste**
- Updated to ForgeOps 7.1
- Using ForgeOps SecretAgent
- Removed `iot` config in favour of an overlay on top of `cdk` config
- Updated config to be 7.1 compatible

### CODE REVIEW

File: deployments/forgeops/overlay/config/7.0/cdk/idm/conf/access.json [hunk 1]
```diff
@@ -280,13 +286,6 @@
       "methods"   : "action",
       "actions"   : "deleteNotificationsForTarget",
       "customAuthz" : "request.additionalParameters.target === (context.security.authorization.component + '/' + context.security.authorization.id)"
-    },
-    {
-      "pattern"   : "managed/*",
```
Comment (LimaEchoCharlie) [inline deployments/forgeops/overlay/config/7.0/cdk/idm/conf/access.json:285]: Just out of curiosity, why was this removed?

**Review by LimaEchoCharlie — APPROVED** [review]
🥇

File: deployments/forgeops/overlay/config/7.0/cdk/idm/conf/access.json (hunk 1 above)
Comment (JacoJooste) [reply deployments/forgeops/overlay/config/7.0/cdk/idm/conf/access.json:285]: Not sure, but the details are here if you want to look into it: https://github.com/ForgeRock/forgeops/commit/fbadcd627738e01b3db40da2a3096bc72fe24afd#diff-1f6d6e19f23d7e0a0824a4d73911d6f57b430727d7400044bb8920284f281661
